src/main/java/org/jabref/logic/importer/fileformat/RisImporter.java

```diff
@@ -216,13 +210,39 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
                             comment = comment + " ";
                         }
                         comment = comment + value;
-                    }
-                    // Added ID import 2005.12.01, Morten Alver:
-                    else if ("ID".equals(tag)) {
+                    } else if ("ID".equals(tag)) {
                         fields.put("refid", value);
                     } else if ("M3".equals(tag) || "DO".equals(tag)) {
                         addDoi(fields, value);
                     }
+                    // fields for which there is no direct mapping in the bibtext standard
+                    else if ("AV".equals(tag)) {
+                        fields.put("archive_location", value);
+                    } else if ("C3".equals(tag)) {
+                        fields.put("event", value);
+                    } else if ("CN".equals(tag) || "VO".equals(tag)) {
+                        fields.put("call-number", value);
+                    } else if ("DB".equals(tag)) {
+                        fields.put("archive", value);
+                    } else if ("N1".equals(tag) || "RN".equals(tag)) {
+                        fields.put("note", value);
+                    } else if ("NV".equals(tag)) {
+                        fields.put("number-of-volumes", value);
+                    } else if ("OP".equals(tag)) {
+                        fields.put("original-title", value);
+                    } else if ("RI".equals(tag)) {
+                        fields.put("reviewed-title", value);
+                    } else if ("RP".equals(tag)) {
+                        fields.put("status", value);
+                    } else if ("SE".equals(tag)) {
+                        fields.put("section", value);
+                    } else if ("ST".equals(tag)) {
+                        fields.put("shortTitle", value);
```

Comment:
`shorttitle` is a standard biblatex field

Reply:
`shorttitle` wasn't even part of `FieldName` so far. I have added it now.